docs/users/intro.md

```diff
@@ -0,0 +1,70 @@
+---
+# BobaBoard Basics
+
+_"BobaBoard will move you and fill you with a feeling of deep peace. I hope your time here will be pleasant and your memories will last forever."_
+
+## What is BobaBoard?
+**The Simple Answer:** A communities website made for fandom by people in fandom.
+
+**The Detailed Answer:** BobaBoard was born as a response to Tumblr's porn ban and to the increasingly hostile nature of online spaces. BobaBoard's main goal (among many) is to create an open source platform, collaboratively built by fans for fans, that can support the needs of small online communities by being adaptable, stable, and fun to use.
+
+While BobaBoard has a unique take on online communities, its inspirations include Tumblr, Discord, LiveJournal, and many others.
+
+## Bobaboard Concepts
+### Realms
+:::info
+
+This section of the guide is still in development! We're still in the process of figuring out how Realms will work. The general idea is explained below, but your input will help shape things!
+
+:::
+
+BobaBoard is made up of indivdual communities where users can interact called **Realms**. Every Realm is a distinct community and users will have to join each Realm where they'd like to have the ability to post and read in. This means that users can't see every post on all of BobaBoard, but can only see the content posted in the Realms they've joined. 
+
+Ultimately, these contained communities will have their own different rules, culture, goals, and user-acceptance criteria, generated by the community members and leaders who are interacting there. Realms may have different approaches to tagging, content moderation, or organization; they may host Realm only events or contests. 
+
+This structure is very similar in different ways to different platforms. It most closely resembles forums, but with modern features; each Realm is it's own forum with subforums to post to. In some ways, it it is similar to Discord servers - Discord is hosting all of the servers, but each server is it's own community. If you're a fandom old (or happen to be familiar), the community structure might also remind you of LJ, DW, or other LJ-clone's communities - except with BobaBoard there's no personal blog.
+
+There are currently two Realms making up BobaBoard:
+- V0: The Alpha
+  > The OG BobaBoard. A closed Alpha for Ms. Boba's friends, Boba volunteers, and early adopters.
+
+- Fandom Coders
+  > The first official "Realm". It's a community of fandom people who also code or want to learn to code, particularly if they use coding in their fandom activities!
+
+For more information on each Realm, visit their sections in the guide.
+
+There will be opportunities in the future for others to establish their own Realms.
+
+### Boards, Not Blogs
+Each **Realm** is made up of multiple Boards. Boards are similar to subforum in a forum or modern Discord channels; they define the broad topics of conversations and give users a heads-up as to what they can expect to find.
```

Comment:
I would maybe say "are similar to `forum sections` or `Discord channels`".

Also, 
> they define the broad topics `of each Realm`, give users a heads-up as to what they can expect to find, and help them avoid or mute topics.